spec/models/room_spec.rb

```diff
@@ -3,7 +3,32 @@
 require 'rails_helper'
 
 RSpec.describe Room, type: :model do
-  let!(:room) { create(:room) }
+  let(:room) { create(:room) }
+
+  describe 'validations' do
+    subject { create(:room) }
+
+    it { is_expected.to belong_to(:user) }
+    it { is_expected.to have_many(:recordings).dependent(:destroy) }
```

Comment:
I know this is irrelevant to this PR but to not forget,
In the future we may want to tirgger a recording delete logic (maybe in a job) that will remove recordings both on GL and on BBB.